hack/istio/install-rh-tutorial.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+##############################################################################
+# install-rh-tutorial.sh
+#
+# Installs the Red Hat Istio Tutorial into your cluster.
+#
+# This requires java and maven in your PATH.
+#
+# See --help for more details on options to this script.
+#
+##############################################################################
+
+# ISTIO_DIR is where the Istio download is installed
+# CLIENT_EXE_NAME is going to either be "oc", "kubectl", or "istiooc"
+# RATE is the rate of requests pushed into the mesh via the traffic generator (if enabled)
+# TUTORIAL_DIR is the parent directory where the tutorial files will be git cloned
+ISTIO_DIR=
+CLIENT_EXE_NAME="oc"
+NAMESPACE="tutorial"
+RATE=1
+TUTORIAL_DIR="/tmp"
+
+# process command line args
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -id|--istio-dir)
+      ISTIO_DIR="$2"
+      shift;shift
+      ;;
+    -td|--tutorial-dir)
+      TUTORIAL_DIR="$2"
+      shift;shift
+      ;;
+    -c|--client-exe)
+      CLIENT_EXE_NAME="$2"
+      shift;shift
+      ;;
+    -n|--namespace)
+      NAMESPACE="$2"
+      shift;shift
+      ;;
+    -tg|--traffic-generator)
+      TRAFFIC_GENERATOR_ENABLED="true"
+      shift;
+      ;;
+    -h|--help)
+      cat <<HELPMSG
+Valid command line arguments:
+  -id|--istio-dir <dir>: Where Istio has already been downloaded. If not found, this script aborts.
+  -td|--tutorial-dir <dir>: Where the tutorial files will be git cloned. Default: /tmp
+  -c|--client-exe <name>: Cluster client executable name - valid values are "kubectl" or "oc" or "istiooc"
+  -n|--namespace <name>: Install the demo in this namespace (default: tutorial)
+  -tg|--traffic-generator: Install Kiali Traffic Generator
+  -h|--help : this message
+HELPMSG
+      exit 1
+      ;;
+    *)
+      echo "Unknown argument [$key]. Aborting."
+      exit 1
+      ;;
+  esac
+done
+
+if [ "${ISTIO_DIR}" == "" ]; then
```

Comment:
This is just nitpicking, but wouldn't it be easier to skip the empty ``ISTIO_DIR`` definition and use ``if [ -z $ISTIO_DIR ]`` here? Or is it necessary to remove the definition is someone has set one in the shell?